client/src/assets/icons/help.css

```diff
@@ -0,0 +1,21 @@
+.wrapper {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  position: relative;
+  padding: inherit;
+}
+
+.speechBubbleIcon {
+  position: absolute;
+  width: 4vh;
+  height: 60vh;
```

Comment:
These can't be in `vh`, they need to be in a proportional unit like `rem`. Otherwise, as you zoom in they will not grow accordingly. This is the Help button at 400% zoom:
![zoomed-help-button](https://user-images.githubusercontent.com/5926192/233444515-2cad922b-b376-44ec-92a8-fe4aa4bac728.png)